account_analytic_account_adjust_display_name/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    analytic_account_display_only_name = fields.Boolean(
```

Comment:
Just so that the field name doesn't go unmatched in case the requirement changes.
```suggestion
    adjust_analytic_account_display_name = fields.Boolean(
```